src/Icons/src/Iconify.php

```diff
@@ -90,6 +91,7 @@ public function fetchIcon(string $prefix, string $name): Icon
 
         return new Icon($data['icons'][$name]['body'], [
             'viewBox' => \sprintf('0 0 %s %s', $width ?? $height, $height ?? $width),
+            'xmlns' => self::XMLNS_URL,
```

Comment:
`xmlns` attribute is usually the first attribute of `<svg>`, let's do the same:
```suggestion
            'xmlns' => self::XMLNS_URL,
            'viewBox' => \sprintf('0 0 %s %s', $width ?? $height, $height ?? $width),
```